src/qibolab/backends.py

```diff
@@ -50,6 +51,25 @@ def __init__(self, platform):
         }
         self.compiler = Compiler.default()
 
+    @property
+    def qubits(self) -> list[Union[str, int]]:
+        return list(self.platform.qubits.keys())
+
+    @property
+    def connectivity(self) -> list[tuple[Union[str, int], Union[str, int]]]:
+        return list(self.platform.pairs)
+
+    @property
+    def natives(self) -> list[str]:    
```

Comment:
```suggestion
    def natives(self) -> list[str]:
```

It's pretty much irrelevant, but this is just a remainder to run `pre-commit install` even in your local clone of the Qibolab repo.